usaspending_api/disaster/v2/views/elasticsearch_account_base.py

```diff
@@ -110,8 +110,10 @@ def build_elasticsearch_search_with_aggregations(self) -> Optional[AccountSearch
         )
         sum_covid_outlay = A(
             "sum",
-            field="financial_accounts_by_award.gross_outlay_amount_by_award_cpe",
-            script={"source": "doc['financial_accounts_by_award.is_final_balances_for_fy'].value ? _value : 0"},
+            script="""doc['financial_accounts_by_award.is_final_balances_for_fy'].value ? (
+             ( doc['financial_accounts_by_award.gross_outlay_amount_by_award_cpe'].size() > 0 ? doc['financial_accounts_by_award.gross_outlay_amount_by_award_cpe'].value : 0)
```

Comment:
Checking the .size() for each field looks rough. Is this the recommended way to handle nulls in aggregate fields? Couldn't `outlay_sum` be used because the ETL is storing all of the values in the single field?

Reply:
checking .size() is the way the elasticsearch docs recommend to check for existing data. We can't use outlay sum due to an edge case - if we have an award with multiple DEFC, and we filter by just "L", for example, we can't use the outlay_sum without including the data from other DEFC.

Reply:
Is `outlay_sum` used anywhere?

Reply:
Looks like it is not